ngraph/core/include/ngraph/op/if.hpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+
+#include "ngraph/function.hpp"
+#include "ngraph/op/parameter.hpp"
+#include "ngraph/op/util/multi_subgraph_base.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v7
+        {
+            /// \brief  write something
+            class NGRAPH_API If : public util::MultiSubGraphOp
+            {
+            public:
+                static constexpr NodeTypeInfo type_info{"If", 7};
+                static constexpr char then_body_index = 0;
+                static constexpr char else_body_index = 1;
+
+                const NodeTypeInfo& get_type_info() const override { return type_info; }
+                bool visit_attributes(AttributeVisitor& visitor) override;
+                If(const Output<Node>& execution_condition);
+                If();
+                explicit If(const OutputVector& values);
+
+                std::shared_ptr<Node>
+                    clone_with_new_inputs(const OutputVector& new_args) const override;
+                std::shared_ptr<Function> get_then_body() const
```

Comment:
Please add a documentation for the operation and public methods.

Reply:
done